examples/flower-client-authentication/README.md

```diff
@@ -0,0 +1,90 @@
+# Flower Client Authentication with PyTorch 🧪
+
+> 🧪 = This example covers experimental features that might change in future versions of Flower
+> Please consult the regular PyTorch code examples ([quickstart](https://github.com/adap/flower/tree/main/examples/quickstart-pytorch), [advanced](https://github.com/adap/flower/tree/main/examples/advanced-pytorch)) to learn how to use Flower with PyTorch.
+
+The following steps describe how to start a long-running Flower server (SuperLink) and a long-running Flower client (SuperNode) with client authentication enabled.
+
+## Preconditions
+
+Let's assume the following project structure:
```

Comment:
Let's walk them through the `git clone` process, like how we do with other examples:

Start by cloning the example project. We prepared a single-line command that you can copy into your shell which will checkout the example for you:
``` bash
git clone --depth=1 https://github.com/adap/flower.git _tmp && mv _tmp/examples/flower-client-authentication . && rm -rf _tmp && cd flower-client-authentication
```
This will create a new directory called `flower-client-authentication` with the following project structure: